content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -106,7 +109,45 @@ The max number of pods that is created within a single polling period. If there
 * **Target Average Value:** The number of messages that will be consumed on a job. It is defined on the scaler side. e.g. `queueLength` on `Azure Storage Queue` scaler.
 * **Running Job Count:** How many jobs are running.
 * **Number of the Scale:** The number of the job that is created.
+```yaml
+  scalingStrategy: "accurate"                 # Optional. Default: default Which ScalingStrategy to use. 
```

Comment:
Minor nit: I'd use 
`scalingStrategy: "default"` and not `accurate` as it is not the default one.  


And fix to comment please:
`# Optional. Default: default. Which Scaling Strategy to use. `
(added . behind `default` and used space in `Scaling Strategy` )

Reply:
Done